lib/cmd/fh3/admin/mbaas/list.js

```diff
@@ -12,8 +12,10 @@ module.exports = {
   'describe': {},
   'url': '/api/v2/mbaases',
   'method': 'get',
-  'postCmd': function(params, cb) {
-    params._table = common.createTableForMbaasTargets(params);
+  'postCmd': function(argv, params, cb) {
+    if (!argv.json) {
```

Comment:
@camilamacedo86 There is small backward incompatibility introduced here. Previously it was possible to programatically call the commands that don't require any input arguments without any arguments, eg.: `fh.admin.mbaas.list()`. With this PR it is no longer possible due to `TypeError: Cannot read property 'json' of undefined`
Although it is relatively simple to fix this in caller code, eg: `fhc.admin.mbaas.list({'json':'json'})`, I would prefer to avoid this backward incompatibility. For instance `if (!argv || !argv.json) { ...` whould do the trick.

This needs to be done not only for mbaas list but for every command that has no required arguments.